packages/react/src/components/ListBox/next/ListBoxTrigger.js

```diff
@@ -36,6 +36,7 @@ const ListBoxTrigger = ({ isOpen, translateWithId: t, ...rest }) => {
     <button
       {...rest}
       aria-label={description}
+      title={description}
```

Comment:
I think for the trigger we removed `title` based on one of Carolyn's suggestions. It seems like having both would basically repeat the same information twice for a screen reader user. I think an alternative she suggested was to use only `title` if we need to offer it (remove the `aria-label in this case). This was the comment IIRC: https://github.com/carbon-design-system/carbon/issues/6938#issuecomment-706717135

Reply:
> it is ok to keep the `<title>Close menu</title>` for sighted mouse users, but please change it to `<title>Close</title>` (which is less verbose, and removes the confusing word "menu")

are you referring to this point?

on a side note, she and I briefly discussed this for #4185 / #8090 and from testing the combobox, multiselect, number input, etc it doesn't seem to be double speaking (in VO at least)

![image](https://user-images.githubusercontent.com/8265238/111223109-b571ca00-85aa-11eb-9785-17cdbee33124.png)

the current multiselect places the title under the SVG rather than the button itself, so that may be an option as well but right now it doesn't seem like either approach is causing VO to read twice

Reply:
@emyarod I'm not sure which screen reader it shows up in, just was going off of what Carolyn shared. Just was sharing that if we needed the `title` then we could just drop the `aria-label` altogether based on what she shared.